financial-connections/src/main/java/com/stripe/android/financialconnections/features/institutionpicker/InstitutionPickerViewModel.kt

```diff
@@ -89,43 +95,43 @@ internal class InstitutionPickerViewModel @Inject constructor(
         onAsync(
             InstitutionPickerState::payload,
             onSuccess = { payload ->
-                eventTracker.track(PaneLoaded(Pane.INSTITUTION_PICKER))
+                eventTracker.track(PaneLoaded(PANE))
                 eventTracker.track(
                     FeaturedInstitutionsLoaded(
-                        pane = Pane.INSTITUTION_PICKER,
+                        pane = PANE,
                         duration = payload.featuredInstitutionsDuration,
                         institutionIds = payload.featuredInstitutions.data.map { it.id }.toSet()
                     )
                 )
             },
             onFail = {
-                eventTracker.logError(
+                handleError(
                     extraMessage = "Error fetching initial payload",
                     error = it,
-                    pane = Pane.INSTITUTION_PICKER,
-                    logger = logger
+                    pane = PANE,
+                    displayErrorScreen = true
                 )
             }
         )
         onAsync(
             InstitutionPickerState::searchInstitutions,
             onFail = {
-                eventTracker.logError(
+                handleError(
                     extraMessage = "Error searching institutions",
                     error = it,
-                    pane = Pane.INSTITUTION_PICKER,
-                    logger = logger
+                    pane = PANE,
+                    displayErrorScreen = false // don't show error screen for search errors.
```

Comment:
Do we inform the user in _any_ way, like with an empty state inside the search screen?

Reply:
yes! https://github.com/stripe/stripe-android/blob/0a35faf4be9c104970025792c155bd3b8e6ee8db/financial-connections/src/main/java/com/stripe/android/financialconnections/features/institutionpicker/InstitutionPickerScreen.kt#L252